src/screens/main_screen.rs

```diff
@@ -87,13 +87,12 @@ impl DrawableComponent for MainScreenLayer {
                 EventResult::Consumed
             }
             KeyCode::Char('s') => {
-                app.task_store.tasks.sort_by_key(|t| t.priority);
+                app.task_store.sort();
                 EventResult::Consumed
             }
             KeyCode::Char('S') => {
-                // NOTE: We can be unstable because `reverse()` will change the order anyway
-                app.task_store.tasks.sort_unstable_by_key(|t| t.priority);
-                app.task_store.tasks.reverse();
+                app.task_store.auto_sort ^= true; // Invert
```

Comment:
```suggestion
                app.task_store.auto_sort = !app.task_store.auto_sort;
```
I think this makes it much easier to understand what is happening.

Reply:
I don't like code repetition ;)
But yeah, that's fair